### PR TITLE
Fix working dir detection

### PR DESCRIPTION
**Pull request by chocoelho**
See: https://github.com/PyCQA/prospector/issues/257#issuecomment-409510013

@carlio how do you feel about it?

### CODE REVIEW

**Comment by chocoelho** [comment]
Tests are broken but first wanna check the idea of considering the rootpath into the minimal path method.

File: prospector/finder.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def get_minimal_syspath(self, absolute_paths=True):
             if dirname not in packages:
                 module_list.append(dirname)
 
-        full_list = sorted(set(module_list) | package_list, key=len)
+        full_list = sorted(set(module_list) | package_list | {self.rootpath}, key=len)
```
Comment (carlio) [inline prospector/finder.py]: I feel like the `module_list` should already include python files from `cwd` but I'm not sure about that. I'll have a play around tomorrow to see why that isn't already happening.

File: tests/finder/test_file_finder.py [hunk 2]
```diff
@@ -43,7 +44,7 @@ def test_long_path_not_a_venv(self):
         Windows doesn't allow extremely long paths. This unit test has to be
         run in Windows to be meaningful, though it shouldn't fail in other
         operating systems.
-        """
+j        """
```
[the remark this answers is not in the document]
Comment (chocoelho) [reply tests/finder/test_file_finder.py]: Thanks for noticing that! Just removed.

File: prospector/finder.py (hunk 1 above)
Comment (chocoelho) [reply prospector/finder.py]: any news on that @carlio ?

**Pull request merged**